Bots/DotNet/Consumers/Composer/ComposerHostBotDotNet/settings/appsettings.json

```diff
@@ -0,0 +1,147 @@
+{
+  "MicrosoftAppId": "",
+  "skill": {
+    "echoSkillBotDotNet": {
+      "endpointUrl": "http://localhost:35400/api/messages",
```

Comment:
I think the only one missing is the Composer Echo

Reply:
- Remove echoSkillBotDotNet21
- Remove EchoSkillBotComposer
- Move waterfallSkillBotDotNet, waterfallSkillBotJS, waterfallSkillBotPython below the echoSkillBotPython
- Add composerSkillBotDotNet below the waterfallSkillBotPython